docs/user-manuals/persistentpodstate.md

```diff
@@ -0,0 +1,156 @@
+---
+title: PersistentPodState
+---
+
+**FEATURE STATE:** Kruise v1.2.0
+
+In Kubernetes, Pod is the "smallest unit of immutable infrastructure", and any change in configuration or environment (e.g., business release, node exception eviction, machine vacations, etc.) will cause the Pod to be rebuilt.
```

Comment:
This paragraph is not necessary here and we can start directly with discussion about statefulset. , we can emphasize that native statefulset can persistent only limited pod state such as pod name, persistent volume, but can not cover other states